cpp/turbodbc/Library/src/time_helpers.cpp

```diff
@@ -5,80 +5,131 @@
 #endif
 #include <cstring>
 #include <sql.h>
-
-#include <boost/date_time/gregorian/gregorian_types.hpp>
-#include <boost/date_time/posix_time/posix_time_types.hpp>
+#include <chrono>
 
 namespace turbodbc {
 
-namespace {
-    boost::posix_time::ptime const timestamp_epoch({1970, 1, 1}, {0, 0, 0, 0});
-    boost::gregorian::date const date_epoch(1970, 1, 1);
-}
+// Alias a time_point with microsecond precision.
+using sys_time_us = std::chrono::time_point<std::chrono::system_clock, std::chrono::microseconds>;
 
+// Define the epoch for timestamps and dates.
+constexpr sys_time_us timestamp_epoch = 
+    sys_time_us(std::chrono::time_point_cast<std::chrono::microseconds>(
+        std::chrono::sys_days{std::chrono::year{1970}/1/1}
+    ));
+constexpr std::chrono::sys_days date_epoch = std::chrono::sys_days{std::chrono::year{1970}/1/1};
```

Comment:
Unless there's another reason to manually encode the epoch, the default constructor of time_point [does this for you](https://en.cppreference.com/w/cpp/chrono/time_point/time_point).

Reply:
Updated